example/reinforcement-learning/a3c/a3c.py

```diff
@@ -139,7 +144,7 @@ def train():
             module.save_params('%s-%04d.params'%(save_model_prefix, epoch))
 
 
-        for _ in range(epoch_size/args.t_max):
+        for _ in range(int(epoch_size/args.t_max)):
```

Comment:
Using "for _ in range(epoch_size//args.t_max):" instead of typecast does not fix the issue here.

Reply:
why?

Reply:
<img width="774" alt="screen shot 2017-12-14 at 3 26 44 pm" src="https://user-images.githubusercontent.com/30911248/34019379-7141471c-e0e3-11e7-9348-080687e654de.png">

 `//` floors the value of the division but the variable type is still float. Range requires the input type to be an integer. 
Is there a performance concern in using type conversion? If yes, I can try and find another fix.